samples/hydra_doc_sample.py

```diff
@@ -104,50 +77,50 @@
                     "readonly": "true",
                     "required": "false",
                     "title": "id",
-                    "writeonly": "true"
+                    "writeonly": "true",
                 },
                 {
                     "@type": "SupportedProperty",
                     "property": "",
                     "readonly": "true",
                     "required": "false",
                     "title": "category",
-                    "writeonly": "true"
+                    "writeonly": "true",
```

Comment:
I don't see the point of adding all these commas.

Please stick to [pep8 package](https://pypi.org/project/pep8/) as a reference libary